contracts/DataModel.sol

```diff
@@ -30,6 +30,8 @@ library FrontDoorStructs {
     struct Job {
         uint256 id;
         uint256 bounty;
+        bool status;
```

Comment:
Just a suggestion, I think status is a bit too generalistic meaning. I believe something live isRemoved would read better. But your call it is fine like this as well :)

Reply:
Sure, I have updated it - will make a commit & push now.